listeners/messageCreateHandlers/tulpaHandler.js

```diff
@@ -97,7 +140,7 @@ module.exports = function (bot) {
 
 		// If there's an attachment, handle that separately
 		if (msg.attachments[0]) {
-			if (msg.attachments.reduce((sum, cur) => sum + cur.size) < 8000000) {
+			if (msg.attachments.reduce((sum, cur) => sum + cur.size, 0) < 8000000) {
```

Comment:
This was returning the first object instead of a sum, so this should fix images not getting proxied.